Automattic/wp-calypso
# Stats: Apply new styling to the WordAds earnings table

Fixes #72676.

## Proposed Changes

Before|After
-|-
<img width="1241" alt="image" src="https://user-images.githubusercontent.com/4044428/222255946-95fb53fb-1bb5-4819-b653-4c9d93224b0a.png">|<img width="1249" alt="image" src="https://user-images.githubusercontent.com/4044428/222010210-9b887a96-de2b-45aa-9b7c-c8325d0f54a1.png">
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/4044428/222255499-d971e7f5-92e7-4632-9d22-3f1837569d04.png">|<img width="1080" alt="image" src="https://user-images.githubusercontent.com/4044428/222255467-b76e5b2f-e1d4-4bfb-b843-8e6838acb113.png">

- Applies new styling to the earnings table on the WordAds Stats and WordAds Earn dashboards.
- Moves `HighlightCardSimple` into the component package.
- Removes code from the `WordAdsEarnings` made redundant by the WordAds highlights section.

## Testing Instructions



* Spin up this live branch and ensure that the new styling has been applied to these pages:
    * Stats -> Ads
    * Tools → Earn → View ad dashboard

## Pre-merge Checklist
